ml/README.md

```diff
@@ -8,9 +8,9 @@ keywords: [machine learning, anomaly detection, Netdata ML]
 
 ## Overview
 
-As of [`v1.32.0`](https://github.com/netdata/netdata/releases/tag/v1.32.0), Netdata comes with some ML powered [anomaly detection](https://en.wikipedia.org/wiki/Anomaly_detection) capabilities built into it and available to use out of the box, with minimal configuration required.
+As of [`v1.32.0`](https://github.com/netdata/netdata/releases/tag/v1.32.0), Netdata comes with some ML powered [anomaly detection](https://en.wikipedia.org/wiki/Anomaly_detection) capabilities built into it and available to use out of the box, with zero configuration required (ML was enabled by default in `v1.35.0-29-nightly` in [this PR](https://github.com/netdata/netdata/pull/13158), previously it required a one line config change).
```

Comment:
Do we still need to reference v1.32 - is that even supported by cloud anymore? I think no.

Reply:
yep - a lot of people are on a wide range of versions and so its good to be accurate like this in the technical docs.